### ISSUE TITLE
IndexError in mode

### ISSUE THREAD

**Issue by Akuli** [ThePhilgrim/MantaTail#80]
Similar to #74 

If you register and then type `MODE x` you get

```
Exception in thread Thread-55:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/home/akuli/MantaTail/mantatail.py", line 187, in recv_loop
    call_handler_function(state, user, args)
  File "/home/akuli/MantaTail/commands.py", line 134, in handle_mode
    process_channel_modes(state, user, args)
  File "/home/akuli/MantaTail/commands.py", line 286, in process_channel_modes
    if args[1][0] not in ["+", "-"]:
IndexError: list index out of range
```

**Comment by ThePhilgrim** [comment]
@Akuli It seems like this bug cannot be reproduced in the way you describe it. Could you please make this issue reproducible?


![Skärmavbild 2022-01-23 kl  17 08 21](https://user-images.githubusercontent.com/76887896/150687446-cebedb4f-d9db-48d5-b619-0cb538cd4a67.png)

![Skärmavbild 2022-01-23 kl  17 07 52](https://user-images.githubusercontent.com/76887896/150687447-690a879d-4ac5-4fd5-a749-c014600f4cdf.png)

**Comment by Akuli** [comment]
Try `MODE #foo` instead of `MODE x`.

**Comment by Akuli** [comment]
Fixed and tested in #93
